packages/darwinia-ui/src/components/Button/styles.scss

```diff
@@ -44,4 +44,5 @@
 
 .dw-btn-spinner-custom-mask {
   background-color: rgb(0 0 0 / 40%) !important;
+  cursor: not-allowed;
```

Comment:
Will this style affect the loading (large loading) here? The large loading does not need to `cursor: not-allowed`

<img width="1431" alt="image" src="https://user-images.githubusercontent.com/21007682/196840095-67c29a4c-3dcb-4848-a5fd-2223073a24d7.png">

Reply:
No it won't. This behavior is only added on the Button component's spinner as you can see from the commit on file path packages/darwinia-ui/src/components/Button/styles.scss

Reply:
Please let me know if further clarification is needed. Thanks